index.html

```diff
@@ -18,7 +18,47 @@
 <body>
   <div class="ellipse-one"></div>
   <div class="ellipse-two"></div>
+  <!-- <div class="popup">
+    <div class="ellipse-one"></div>
+    <div class="ellipse-two"></div>
+    <div class="popup-box">
+      <div class="overlay"></div>
+      <div class="column">
+        <div class="popup-header"><h3 class="popup-title">Base Apparel coming soon page </h3> <div class="vide" >&times;</div></div>
+        <div class="pop-image"></div>
+      </div>
+      <div class="pop-container">
+        <p class="about-popup"> A daily selection of privately
+          personalized reads; no accounts or sign-ups
+          required. has been the industry's standard
+          dummy text ever since the 1500s, when an
+          unknown printer took a standard dummy
+          text.</p>
+        <ul class="pop-tools-list  ">
+          <li class="tools-item right-border">
+            <a href="https://github.com/DanUshindi/Base/blob/main/style.css">css</a>
```

Comment:
Great work on making the changes requested by the previous reviewer but please ensure that all commented codes are removed from your codebase. It makes your code clean.